geniushubclient/zone.py

```diff
@@ -138,10 +149,12 @@ def footprint_schedule(raw_json) -> Dict:
 
             return root
 
-        self._data = result = {"id": self._raw["iID"], "name": self._raw["strName"]}
+        self._data = result = {}
         raw_json = self._raw  # TODO: remove raw_json, use self._raw
 
         try:  # convert zone (v1 attributes)
+            result = self._properties.populate_v1_data(result)
+
             result["type"] = ITYPE_TO_TYPE[raw_json["iType"]]
```

Comment:
It seems some of this code is duplicated in the `_properties` class?

Reply:
Yep, good spot.